dev_docs/cxg.md

```diff
@@ -43,25 +40,25 @@ CXG consumers (readers) MUST be prepared to handle any legal TileDB compression,
 ### CXG
 
 The CXG is a TileDB group containing all data and metadata for a single annotated matrix.  The following objects MUST be present in a CXG, except where noted as optional:
-* __obs__: a TileDB array, of shape (n_obs,), containing obs annotations, each annotation stored in a separate TileDB array attribute.
-* __var__: a TileDB array, of shape (n_var,), containing var annotations, each annotation stored in a separate TileDB array attribute.
-* __X__: a TileDB array, of shape (n_obs, n_var), with a single TileDB attribute of numeric type.
-* __X_col_shift__: (optional) TilebDB Array used in column shift encoding, shape (n_var,), dtype = X.dtype. Single unnamed numeric attribute.  
-* __emb__: a TileDB group, which in turn contains all (zero or more) embeddings.
-* __emb__/\<embedding_name\>__: a TileDB array, with a single anonymous attribute, of numeric type, and shape (n_obs, N>=2).
-* __cxg_group_metadata__: an empty TileDB array, used to store CXG-wide metadata
+* `__obs__`: a TileDB array, of shape (n_obs,), containing obs annotations, each annotation stored in a separate TileDB array attribute.
```

Comment:
This quoting changes the schema definition entirely -- `obs` becomes `__obs__`.  I assume this is unintentional, and you want to drop the bolding (__) from each, or revert the back-tick-quoting.

Reply:
indeed, I misinterpreted `__` as part of the literal names. fixed!